[2025-04-08T19:14:28.316+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-08T19:14:28.362+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: init_elt_pipeline.create_bucket manual__2025-04-08T19:14:25.645127+00:00 [queued]>
[2025-04-08T19:14:28.371+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: init_elt_pipeline.create_bucket manual__2025-04-08T19:14:25.645127+00:00 [queued]>
[2025-04-08T19:14:28.372+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-08T19:14:28.389+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_bucket> on 2025-04-08 19:14:25.645127+00:00
[2025-04-08T19:14:28.399+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=291) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-08T19:14:28.401+0000] {standard_task_runner.py:63} INFO - Started process 303 to run task
[2025-04-08T19:14:28.400+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'init_elt_pipeline', 'create_bucket', 'manual__2025-04-08T19:14:25.645127+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/init_elt_pipeline.py', '--cfg-path', '/tmp/tmp0qvued4w']
[2025-04-08T19:14:28.403+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask create_bucket
[2025-04-08T19:14:28.425+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-08T19:14:28.474+0000] {task_command.py:426} INFO - Running <TaskInstance: init_elt_pipeline.create_bucket manual__2025-04-08T19:14:25.645127+00:00 [running]> on host 01b011050d21
[2025-04-08T19:14:28.566+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='init_elt_pipeline' AIRFLOW_CTX_TASK_ID='create_bucket' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T19:14:25.645127+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T19:14:25.645127+00:00'
[2025-04-08T19:14:28.567+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-08T19:14:28.590+0000] {init_minio.py:24} INFO - Minio client connected successfully
[2025-04-08T19:14:28.602+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-08T19:14:28.602+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-08T19:14:28.616+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=init_elt_pipeline, task_id=create_bucket, run_id=manual__2025-04-08T19:14:25.645127+00:00, execution_date=20250408T191425, start_date=20250408T191428, end_date=20250408T191428
[2025-04-08T19:14:28.655+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-08T19:14:28.688+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-08T19:14:28.692+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
